README.md

```diff
@@ -26,8 +24,20 @@ If you wish to scan all namespaces in your cluster, remove the `--exclude-namesp
 
 <img src="docs/summary.png">
 
+# Being part of the team
+
+We invite you to our team! We are excited about this project and want to return the love we get. 
+
+Want to contribute? Want to discuss something? Have an issue?
+
+* Open a issue, we are trying to respond within 48 hours
+* [Join us](https://discordapp.com/invite/CTcCaBbb) in a discussion on our discord server! 
+
+[<img src="docs/discord-banner.png" width="100" alt="logo" align="center">](https://discordapp.com/invite/CTcCaBbb)
```

Comment:
This is the invite link, what is the logo?

Reply:
Both and both (I want to have the logo on our page, which is also clickable)

Reply:
Oh, I dont see you added the image to the docs dir...